docs/content/release/0.14.0-notes.rst

```diff
@@ -0,0 +1,406 @@
+        rhoMap=mapping
+    )
+
+    # Compute the fields from `sim`
+    fields = sim.fields(model)
+    # Predict data also using `sim`
+    dpred = sim.dpred(model, f=fields)
+
+See? The ``Simulation`` handles predicting data. This change will also make it
+easier to interface with external codes for inversion purposes, as all that is
+needed to be defined to use a ``Simulation`` for an ``InvProblem``, is
+``sim.dpred``, ``sim.Jvec`` and ``sim.Jtvec``.
+
+Please see the documentation for the :class:`SimPEG.simulation.BaseSimulation` class
+as well as the individual methods' ``Simulation``-s, for a detailed description of
+arguments, but largely it accepts the same arguments as the ``Problem`` class,
+but now also requires a ``Survey`` to be set.
+
+The ``Data`` class
+------------------
+Previously, field data would also live within the ``Survey`` class. Which was not
+only confusing, but placed the importance on the wrong component. When inverting geophysical
+data, we are concerned with the data. Thus we would like to enforce this importance
+by making data live in a dedicated ``Data`` class. This ``Data`` class can act like a smart
+dictionary to grab data associated with a specific source, receiver combination.
+More importantly, this ``Data`` class is where we store information related to observed
+data and its errors. This class started in the ``SimPEG.Survey`` module, but has
+now been moved into its own new module ``SimPEG.data``. See the documentation for
+the :class:`SimPEG.data.Data` for all of the details.
+
+Previously,
+
+.. code-block:: python
+
+    # Add data to the survey
+    survey.dobs = dobs
+    survey.std = 0.05  # a 5% relative error
+    survey.eps = 1.0E-6  # a noise floor
+
+Now,
+
+.. code-block:: python
+
+    # Create a data object
+    data = data.Data(dobs=dobs, relative_error=0.05, noise_floor=1e-6)
+
+You might also notice that we changed the name of the terms used to construct
+the standard deviation. See issue `#846 <https://github.com/simpeg/simpeg/issues/846>`__.
+Previously ``survey.std`` represented an error that was relative to the absolute
+value of the data. The name of this term is misleading, as it is not actually
+the classic statistical standard deviation.
+
+Previously the uncertainty was constructed as:
+
+.. code-block:: python
+
+    uncertainty = survey.std * np.abs(survey.dobs) + survey.eps
+
+We now have updated the names to be clearer and more in line with what we would
+naturally expect, which is accessed from `data.standard_deviation`. The value
+that is returned from this is now defined as:
+
+.. code-block:: python
+
+    data.standard_deviation = (
+        data.relative_error * np.abs(data.dobs) +
+        data.noise_floor
+    )
+
+This ``Data`` class is now also the object that is returned from:
+
+.. code-block:: python
+
+    data = sim.make_synthetic_data(
+        m, relative_error=0.05, noise_floor=0.0, f=None, add_noise=True
+    )
+
+The ``L2DataMisfit`` class
```

Comment:
Do we want this to be just `DataMisfit` ? (I know all that we have at the moment is the L2DataMisfit, but the concept would be the same for any generic data misfit)

Reply:
Are you referring to just the title of this section? If so I'd be happy with that.

Reply:
Yep, just the title 
```suggestion
The ``DataMisfit`` class
```